README.md

```diff
@@ -138,6 +138,19 @@ console.log(data);
 }
 ```
 
+Example using an object:
```

Comment:
[Question] I feel like the object use case is probably a little more real-world than the string one. WDYT about putting it first?

It is nice how this one is a lot fewer lines than the string one. Which IMO makes it even better for going first...

Not a blocker on my end, I'm happy either way.

Reply:
Updated